python/scope.py

```diff
@@ -254,6 +255,74 @@ def get_name_type(self, name_type, err=True, get_type=True):
 		else:
 			return pattern, self.parse_type(name_type.children[1], err) if get_type else 'whatever'
 
+	"""
+	This is to be used to get the NClass value for evaluating classes
+	"""
+	def get_class_val(self, modifiers, name, class_value):
+		scope = Scope()
+		for ins in class_value.children:
+			type_check_class_command(ins, scope)
+		return NClass(None, None, None, None)
+
+		# TODO get type_check_class_command working
```

Comment:
I believe this is no longer used; the class body is just straight up a normal block like function bodies, and it just uses public variables in the class's scope for the methods and properties accessible outside

Reply:
because the class body is the constructor body, so it makes sense to allow other instructions